core/controllers/suggestion.py

```diff
@@ -770,10 +778,19 @@ def get(self, target_type: str, suggestion_type: str) -> None:
                 .get_suggestions_with_editable_explorations(
                     reviewable_suggestions))
         elif suggestion_type == feconf.SUGGESTION_TYPE_ADD_QUESTION:
+            topic_name = self.normalized_request.get('topic_name')
+            topic_skills = None
+            if topic_name is not None and topic_name != 'All':
+                topic = topic_fetchers.get_topic_by_name(topic_name)
+                if topic is None:
+                    raise self.InvalidInputException(
+                        'The supplied topic: %s is not valid'
```

Comment:
Nit: suggest dropping the colon and putting `\'` around the `%s` -- `'The supplied topic \'%s\' is not valid`

Reply:
sg!